### PR TITLE
feat: Controllers logic and import fix

### PR DESCRIPTION
**Pull request by hhrynenko**
Now controllers have right import and more optimised logic.

### CODE REVIEW

File: controllers/commentsController.js [hunk 1]
```diff
@@ -76,14 +98,19 @@ const updateComment = async (req, res) => {
                 error: 'Request body is empty.',
             });
         }
-        const { newCommentText } = req.body;
+        const { commentText } = req.body;
         const id = parseInt(req.params.id, 10);
-        if (id > await City.count()) {
+        const idCheck = await Comment.findOne({
```
Comment (stovpak) [inline controllers/commentsController.js]: rename to const commentInDb or const commentToUpdate

File: controllers/commentsController.js [hunk 2]
```diff
@@ -51,18 +69,22 @@ const addComment = async (req, res) => {
             });
         }
         const { cityName, commentText } = req.body;
-        const commentToSave = allowedCitiesArr.find((city) => city === cityName);
-        if (isEmpty(commentToSave)) {
+        const comment = await City.findOne({
```
Comment (stovpak) [inline controllers/commentsController.js]: rename to const cityInDb or const relateCity

File: controllers/commentsController.js [hunk 3]
```diff
@@ -1,46 +1,64 @@
...
-        return res.status(501).json({
-            error: 'There are no city name.',
+    try {
+        const { cityName } = url.parse(req.url, true).query;
+        if (isEmpty(cityName)) {
+            return res.status(501).json({
+                error: 'There are no city name.',
+            });
+        }
+        const { id } = await City.findOne({
```
Comment (stovpak) [inline controllers/commentsController.js]: get and check full object

Comment (hhrynenko) [reply controllers/commentsController.js]: You mean check if object is not null?

**Pull request merged**